feature-libs/requested-delivery-date/root/components/delivery-mode-date-picker/delivery-mode-date-picker.component.ts

```diff
@@ -48,6 +48,7 @@ export class DeliveryModeDatePickerComponent implements OnInit, OnDestroy {
     requestDeliveryDate: new FormControl(),
   });
   protected isDatePickerReadOnly: boolean = true;
+  protected readonly KEY_DEBOUNCE_TIME = 500;
```

Comment:
Are we using this property somewhere?

Reply:
It was added while testing the debounce changes, will remove it.
thanks